src/balena-model.ts

```diff
@@ -2,6 +2,8 @@
  * This file is auto-generated with `npm run generate-model-types`
  */
```

Comment:
@Page- can we also remove this other one from the generation on the package.json? I don't think it is necessary considering now abstract-sql-to-ts is already putting it in

Reply:
sure